lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
-                              '${row.title}',
-                          outsideLabelStyleAccessorFn: (MatureCount row, _) =>
-                              charts.TextStyleSpec(
-                                  color: charts.MaterialPalette.black),
                         ),
                       ],
-                      animate: true,
                     ),
-                  ),
+                  ],
+                ),
+                Padding(
+                  padding: EdgeInsets.only(bottom: 16),
                 ),
-                Center(
-                    child: Column(
-                  mainAxisAlignment: MainAxisAlignment.center,
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.end,
                   children: <Widget>[
-                    Text(
-                      totalCardsNumber,
-                      overflow: TextOverflow.ellipsis,
-                      style: SarakaTextStyles.body.copyWith(fontSize: 40),
-                      textAlign: TextAlign.center,
-                    ),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      children: <Widget>[
-                        Padding(
-                          padding: EdgeInsets.only(left: 60.0),
+                    FlatButton(
+                      child: Container(
+                        width: 200,
+                        decoration: BoxDecoration(
+                          borderRadius: BorderRadius.all(Radius.circular(20.0)),
                         ),
-                        Text(
-                          'Cards',
-                          overflow: TextOverflow.ellipsis,
-                          style: SarakaTextStyles.body2.copyWith(fontSize: 12),
-                          textAlign: TextAlign.center,
-                        ),
-                      ],
-                    ),
-                    Center(
-                      child: GestureDetector(
                         child: Row(
-                          mainAxisAlignment: MainAxisAlignment.center,
+                          mainAxisAlignment: MainAxisAlignment.end,
                           children: <Widget>[
                             Text(
-                              'See card list',
+                              'SEE ALL ' +
+                                  totalCardsNumber.toString() +
+                                  ' CARDS',
                               overflow: TextOverflow.ellipsis,
-                              style: SarakaTextStyles.headlineSmall,
-                              textAlign: TextAlign.center,
+                              style: SarakaTextStyles.heading.copyWith(
+                                  fontSize: 20, color: SarakaColors.darkGray),
                             ),
                             Icon(
-                              Icons.keyboard_arrow_down,
+                              Icons.arrow_forward_ios,
```

Comment:
Is the feather icon's one looking not good?

Reply:
mmm yeah, I Like feather one.